SoObjects/SOGo/SOGoGCSFolder.m

```diff
@@ -1858,10 +1860,9 @@ - (void) setRoles: (NSArray *) roles
   aUID = uid;
   if (![uid hasPrefix: @"@"])
     {
-      // Prefix the UID with the character "@" when dealing with a group
-      domain = [[context activeUser] domain];
-      group = [SOGoGroup groupWithIdentifier: uid inDomain: domain];
-      if (group)
+      NSDictionary *dict;
+      dict = [[SOGoUserManager sharedUserManager] contactInfosForUserWithUIDorEmail: uid];
```

Comment:
Respect the domain by using `[SOGoUserManager contactInfosForUserWithUIDorEmail:inDomain:]`.

Reply:
Resolved in commit 948c89f